src/components/application_manager/src/app_launch/app_launch_data_json.cc

```diff
@@ -187,15 +202,16 @@ std::vector<ApplicationDataPtr> AppLaunchDataJson::GetAppDataByDevMac(
       }
     }
   }
-
+  accessor.GetMutableData().set_dictionary(dictionary);
```

Comment:
@YarikMamykin its a getter and I don't see that dictionary is modified in this function somehow. Why are you setting it?

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/3dcb90541c1ee0310a396d2564a95eba3e12237e

Reply:
@AKalinich-Luxoft `GetSavedApplicationDataList` **(line: 185)** method modifies the Last State dictionary. Those changes should be saved.